employees/views_business_logic.py

```diff
@@ -0,0 +1,114 @@
+from employees.models import Employee
+
+
+class ContractExtension:
+
+
+    name=None
+    exp_date=None
+
+    def add_one_month(self, employee_id):
+        employee_to_update = Employee.objects.get(pk=employee_id)
+
+        self.name=employee_to_update.first_name+ ' '+employee_to_update.last_name
+
+
+        old_date = employee_to_update.contract_exp_date
+        self.exp_date=old_date
```

Comment:
Please stick to one styling. There should be space between "=" sign unless it's a function call like `.get(pk=employee_id)`.

Reply:
done for the whole file